protobuf/syft_proto/types/torch/v1/script_function.proto

```diff
@@ -2,7 +2,7 @@ syntax = "proto3";
 
 package syft_proto.types.torch.v1;
 
-// torch._C.Function
-message CFunction {
+// torch.jit.ScriptFunction
+message ScriptFunction {
```

Comment:
Same here: could you make `ScriptFunction` a separate proto? We can come back and remove the duplication after we’ve pushed a PySyft release with a more restrictive dependency on `syft-proto`.